startup.sh

```diff
@@ -0,0 +1,13 @@
+cd ../data;
+mkdir db;
```

Comment:
maybe make this `mkdir -p db` so it doesn't fail if the directory already exists?